concordium-consensus/src/Concordium/KonsensusV1/Consensus.hs

```diff
@@ -325,3 +328,62 @@ getProtocolUpdateState = do
                     { puQueuedTime = ts,
                       puProtocolUpdate = pu
                     }
+
+-- |Get the list of rounds that fall within a finalized epoch, together with which baker was
+-- eligible to bake in that round and whether a block in that round was included in the finalized
+-- chain.
+-- This returns 'Nothing' if the epoch is not completely finalized (i.e. there is no finalized
+-- block in a higher round).
+getWinningBakersForEpoch ::
```

Comment:
Why is this function being put here as opposed to `Implementation.hs`. The function isn't part of the consensus protocol, it's a query on the provided `SkovData`.